### PR TITLE
Android support

### PR DESCRIPTION
**Pull request by rubenmoral**

### CODE REVIEW

File: library/src/test/java/com/digi/xbee/api/connection/android/CircularByteBufferWriteTest.java [hunk 1]
```diff
@@ -0,0 +1,274 @@
+/**
+* Copyright (c) 2014 Digi International Inc.,
```
Comment (diescalo) [inline library/src/test/java/com/digi/xbee/api/connection/android/CircularByteBufferWriteTest.java]: Update copyright.

File: library/src/test/java/com/digi/xbee/api/connection/android/CircularByteBufferReadTest.java [hunk 2]
```diff
@@ -0,0 +1,690 @@
+/**
+* Copyright (c) 2014 Digi International Inc.,
```
Comment (diescalo) [inline library/src/test/java/com/digi/xbee/api/connection/android/CircularByteBufferReadTest.java]: Update copyright.

File: library/src/test/java/com/digi/xbee/api/connection/android/CircularByteBufferCreateTest.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
+/**
+* Copyright (c) 2014 Digi International Inc.,
```
Comment (diescalo) [inline library/src/test/java/com/digi/xbee/api/connection/android/CircularByteBufferCreateTest.java]: Update copyright.

**Comment by rubenmoral** [comment]
Waiting to merge until `wlan_support` is merged into `master` in order to add Android constructors to IP device classes.

**Comment by rubenmoral** [comment]
Manually merged with squash.

**Pull request closed without merging**